### PR TITLE
Add page of cloud storage

### PR DESCRIPTION
**Pull request by 97carmine**
Add information about cloud storage and its combination with Rclone and Jellyfin.

### CODE REVIEW

File: docs/media/cloud-storage.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Cloud storage
+
+There is no native support for cloud storage of services such as Amazon S3, Dropbox, Google Drive...
```
Comment (anthonylavado) [inline docs/media/cloud-storage.md]:
```suggestion
There is no native support for cloud storage of services such as Amazon S3, Dropbox, Google Drive.
```

File: docs/media/cloud-storage.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+The way to use your cloud storage with Jellyfin is to use Rclone.
```
Comment (anthonylavado) [inline docs/media/cloud-storage.md]:
```suggestion
One way to use your cloud storage with Jellyfin is to use Rclone.
```

**Comment by anthonylavado** [comment]
Hi @97carmine, are you able to accept the suggestions I’ve made?

**Comment by 97carmine** [comment]
Yes

**Pull request merged**